lib/GraphReader.cc

```diff
@@ -163,54 +165,80 @@ void GraphReader::read_partition(BipartiteGraph::ContainerType& vmap) {
     match(TOK_AT);
     match(TOK_END);
 }
-
 /// @PreferenceLists (A|B)
 /// preference lists
 /// @End
 /// preference lists for a vertex are given in this format
-/// v: a, b, c ;
+/// v: a, (b, c), d ;
```

Comment:
This modifies the previous line, but does not indicate what (b,c) mean. A description of the syntax should be provided.